src/main/resources/reference.conf

```diff
@@ -1197,7 +1200,7 @@ resourceTypes = {
         roleActions = ["read_dataset", "read_data", "read_policies", "link_snapshot"]
       }
       admin = {
-        roleActions = ["share_policy::steward", "read_policies", "alter_policies"]
+        roleActions = ["share_policy::steward", "read_policies", "alter_policies", "view_snapshot_builder_config"]
```

Comment:
Admin should be kept really small (where an admin can up their privs if needed).  Is this a temporary thing while this is in dev.  I think this makes more sense as a steward roleAction, and possible custodian (since it has manage_schema and they seem somewhat related) though fair if we want to leave off custodian

Reply:
Are there stewards in Prod who could use this action? Technically this code isn’t gated to dev, so I’m hesitant to grant it to a role that users could have.

Reply:
We could also make a new role just for this feature until it's ready.

Reply:
There are.  It's basically the owner of a dataset.  I think while we're building it out, it makes sense to do that but I just want to make sure that we eventually add it to stewards (since they'll be the ones interacting in the long run)

Reply:
Eventually, I think actually this action will belong to Reader (or whatever our new "Aggregate data reader" role is) - but steward for now seems okay. I can move it.

Reply:
Oh right. derp, this is the view config, not the change config.  I think it's ok to keep it as admin but either is cool

Reply:
@pshapiro4broad , I'd keep the roles pretty simple so that we don't need to change anything on the TDR side when things are ready to go

Reply:
Totally.  I think it's cool to leave as is